readme/manual-setup.md

```diff
@@ -157,14 +150,21 @@ export SUPERSET_HOSTNAME=analytics-"${IP}.traefik.me";
 #### With Apache 2
 
 ```bash
-docker compose -f docker-compose.yml -f docker-compose-proxy.yml -p $DISTRO_GROUP up
+docker -f docker-compose.yml -f docker-compose-proxy.yml -f docker-compose-senaite.yml -f docker-compose-odoo.yml -f docker-compose-demo.yml -p $DISTRO_GROUP up
 ```
 #### With Traefik
 
 ```bash
-docker compose -p $DISTRO_GROUP up
+docker compose -f docker-compose.yml -f docker-compose-senaite.yml -f docker-compose-odoo.yml -f docker-compose-demo.yml -p $DISTRO_GROUP up
 ```
+The above commands will startup an Ozone instance that will be populated with some demo data depending on whether the environment property `NUMBER_OF_DEMO_PATIENTS` has been set to a given value greater than `0`. Otherwise no demo data is generated.
```

Comment:
Let's leave that whole sentence out, however we need to explain in the section 'How to activate Ozone demo data generation' what is the purpose of `NUMBER_OF_DEMO_PATIENTS`.